ui-admin/src/populate/PopulateControls.tsx

```diff
@@ -24,32 +24,25 @@ export const OverwriteControl = ({ text, isOverwrite, setIsOverwrite }:
 /**
  * renders a control for specifying a file name --this could be updated to handle the conventions of the various paths
  */
-export const useFileNameControl = () => {
-  const [fileName, setFileName] = useState('')
-  return {
-    fileName,
-    fileNameControl: <label className="form-label">
+export const FileNameControl = ({ fileName, setFileName }: {fileName: string, setFileName: (n: string) => void}) => {
+  return <label className="form-label">
             File path (from <code>/populate/src/main/resources/seed</code>)
-      <input type="text" value={fileName} className="form-control"
-        onChange={e => setFileName(e.target.value)}/>
-    </label>
-  }
+    <input type="text" value={fileName} className="form-control"
```

Comment:
Any reason not to use `TextInput` here?

Also, I could see this as a thin wrapper around a `TextInput` and its `label`, in which case its props API could be the same as `TextInput`, specifically directly passing through `value` and `onChange`. It's probably a toss-up, but all else being equal, I try to favor following existing patterns.

Same for `PortalShortcodeControl` below.